### PR TITLE
Update CHANGES.md for edge-18.10.4 release

### PR DESCRIPTION
**Pull request by rmars**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## edge-18.10.4
+* CLI
```
Comment (klingerf) [inline CHANGES.md]: Can you add an extra space between the header and the first bullet here? Also, we should call out the material switchover more prominently. It's a Big Deal!!!

```
## edge-18.10.4

This releases includes a major redesign of the web frontend to make use of the
Material design system. Additional features that leverage the new design are
coming soon. This release also includes the following changes:

* CLI
  * **Fixed** Fixed an issue with the `--registry` install flag not accepting
    hosts with ports (thanks, @alenkacz!)
* Web UI
  * **New** Added a new Grafana dashboard for authorities (thanks, @alpeb!)
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+  * **New** adds the concept of service profiles to the controller
```
Comment (wmorgan) [inline CHANGES.md]: Is there a user-facing impact of this change that we can describe?

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+* Web UI
+  * **New** Revamped the look of the dashboard by switching component libraries
```
Comment (wmorgan) [inline CHANGES.md]: How about: Major look and feel update to the Linkerd dashboard by switching component libraries from antd to material-ui

File: CHANGES.md (hunk 2 above)
Comment (rmars) [reply CHANGES.md]: Yeah, I think we should hold off on the bullet point about service profiles until we have a user-facing feature!

**Pull request merged**